holidayextras/ui-toolkit
# Move list style to UL and removed padding

### :bug: **BUG**

#### What does this PR do? (please provide any background)
- `list-style` should not be added to an `li` it should be added to the `ul` is it conatiner within
- I have also removed default padding from the icon list
- This is because on the docs (with bootstrap) everything looked okay, but in reality it wasn't quite right

#### What tests does this PR have?
None

#### How can this be tested?
- Clone branch
- run `npm run docs` and check that the lost icon looks aligned and new styles are present (see screenshot)

#### Screenshots / Screencast
<img width="1346" alt="screen shot 2015-11-23 at 14 22 25" src="https://cloud.githubusercontent.com/assets/778942/11338726/a70654d8-91ed-11e5-80d4-6b1945f213b6.png">


#### What gif best describes how you feel about this work?
![Head BANG](https://media.giphy.com/media/xTiTnslZ0E5sqMbEac/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
